pkg/cmd/root/root.go

```diff
@@ -97,6 +98,7 @@ func NewCmdRoot(f *cmdutil.Factory, version, buildDate string) *cobra.Command {
 	cmd.AddCommand(extensionCmd.NewCmdExtension(f))
 	cmd.AddCommand(searchCmd.NewCmdSearch(f))
 	cmd.AddCommand(secretCmd.NewCmdSecret(f))
+	cmd.AddCommand(variableCmd.NewCmdVariable(f))
```

Comment:
Should this be added to the group of commands for actions?

```
gh actions
```

Reply:
The command is following the `secrets` model of implementation, variables, in the future may support more integrations(like secrets does for `dependabot` and `codespaces` today along with actions), therefore did not want to tie is to actions integrations only.

Reply:
Thank you for that additional insight! 🙇 

I originally presumed that a command might be able to live in multiple groups, however I'm not 💯 sure there as looking at [`workflow` base command](https://github.com/cli/cli/blob/88711201ae886b45ebab320a6cb5ff72f681854a/pkg/cmd/workflow/workflow.go#L13-L19) shows a singular group.

https://github.com/cli/cli/blob/88711201ae886b45ebab320a6cb5ff72f681854a/pkg/cmd/workflow/workflow.go#L13-L19

All that to say, I've got nothing additional to share as a CLI groupie ❤️

Reply:
> Should this be added to the group of commands for actions?
> 
> ```
> gh actions
> ```